modules/AOS_PDF_Templates/samples/smpl_Account_Sample.php

```diff
@@ -5,13 +5,15 @@ function getType() {
 		}
 
 		function getBody() {
-		global $locale;
-		$d_image = explode('?',SugarThemeRegistry::current()->getImageURL('company_logo.png'));
+        global $locale, $sugar_config;
+        $baseUrl = $sugar_config['site_url'];
+        $logoUrlArr = explode('?',SugarThemeRegistry::current()->getImageURL('company_logo.png'));
+        $logoUrl = $baseUrl . '/' . $logoUrlArr[0];
```

Comment:
Can you create a new service class for the logoUrl?

Reply:
Updated